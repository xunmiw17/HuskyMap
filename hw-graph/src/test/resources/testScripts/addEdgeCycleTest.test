# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add three edges forming a cycle
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n3 e2
AddEdge graph1 n3 n1 e3

# Print the nodes in the graph and the outgoing edges from all three nodes
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3