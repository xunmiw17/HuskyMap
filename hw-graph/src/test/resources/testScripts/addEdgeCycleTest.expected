# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add three edges forming a cycle
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n3 in graph1
added edge e3 from n3 to n1 in graph1

# Print the nodes in the graph and the outgoing edges from all three nodes
graph1 contains: n1 n2 n3
the children of n1 in graph1 are: n2(e1)
the children of n2 in graph1 are: n3(e2)
the children of n3 in graph1 are: n1(e3)